content/en/serverless/azure_app_services.md

```diff
@@ -43,7 +43,9 @@ The Datadog extension for Azure App Services provides additional monitoring capa
 
 2. The extension only supports Azure App Service Web Apps. **Function Apps are not supported**.
 
-3. The Datadog .NET APM extension supports the following .NET runtimes in both x64 and x86 architectures when running on Windows instances (AAS does not yet support extensions on Linux). For more details about automatically instrumented libraries, see the [Tracer documentation][9].
+<div class="alert alert-warning">To get notified when a private beta is available for Function Apps or additional runtimes, [sign up here][9].</div>
```

Comment:
Need to change markdown style link to html-style link (see my comments in Jira card).

Reply:
Thanks, @kayayarai I think this should be fixed.